hystrix-contrib/hystrix-codahale-metrics-publisher/src/test/java/com/netflix/hystrix/contrib/codahalemetricspublisher/HystrixCodaHaleMetricsPublisherCommandTest.java

```diff
@@ -20,27 +23,28 @@ public void setup() {
         HystrixPlugins.getInstance().registerMetricsPublisher(new HystrixCodaHaleMetricsPublisher(metricRegistry));
     }
 
-    @After
-    public void teardown() {
-        HystrixPlugins.reset();
+    @Test
+    public void testCommandSuccess() throws InterruptedException {
+        Command command = new Command();
+        command.execute();
+
+        Thread.sleep(1000);
```

Comment:
I could, but then I introduce an extra dependency for this specific test.  You could argue that a utility like this should be applied across all of the unit tests.  My personal opinions is that the current state isn't enhanced by that much by a wholesale change to this style, but I'm happy to be convinced otherwise